src/main/java/com/revealprecision/revealserver/messaging/message/PersonBusinessStatusAggregate.java

```diff
@@ -0,0 +1,49 @@
+package com.revealprecision.revealserver.messaging.message;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+
+@Setter
```

Comment:
@Data is equivalent to  @Getter @Setter @RequiredArgsConstructor @ToString @EqualsAndHashCode, so you can remove @Gettet and @Setter depending on your goal.